medical-portal/src/API/RSBC.DMF.MedicalPortal.API/Model/DocumentFactory.cs

```diff
@@ -0,0 +1,50 @@
+﻿using Google.Protobuf.WellKnownTypes;
+using Pssg.SharedUtils;
+using Rsbc.Dmf.CaseManagement.Service;
+
+
+namespace RSBC.DMF.MedicalPortal.API.Model
+{
+    public class DocumentFactory
```

Comment:
let's move this to SharedUtils

Reply:
you can use this as a reference, I used this to verify we don't still have pipeline errors when using SharedUtils in cms-adapter https://github.com/bcgov/rsbc-dmf/pull/1625/files